<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Byz Ison</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_vu2">
        <property name="text">
         <string>Vu'</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_pa2">
        <property name="text">
         <string>Pa'</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ni2">
        <property name="text">
         <string>Ni'</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_zo">
        <property name="text">
         <string>Zo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ke">
        <property name="text">
         <string>Ke</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_di">
        <property name="text">
         <string>Di</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ga">
        <property name="text">
         <string>Ga</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_vu">
        <property name="text">
         <string>Vu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_pa">
        <property name="text">
         <string>Pa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ni">
        <property name="text">
         <string>&lt;Ni&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_zo0">
        <property name="text">
         <string>Zo,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ke0">
        <property name="text">
         <string>Ke,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_di0">
        <property name="text">
         <string>Di,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ga0">
        <property name="text">
         <string>Ga,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Diatonic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Enharmonic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Soft Chromatic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard Chromatic</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
